Add select_related support to Polymorphic models

szok opened the pull request:
Adding support for select_related in the Django admin change lists for
models that use PolymorphicQuerySet

mkurek commented:
Couldn't you "somehow" use `polymorphic_select_related` just like in API (for `BaseObject`)? 

https://github.com/allegro/ralph/blob/ng/src/ralph/api/utils.py
https://github.com/allegro/ralph/blob/ng/src/ralph/lib/polymorphic/models.py#L59

Commit:
Add select_related support to Polymorphic models

Adding support for select_related in the Django admin change lists for
models that use PolymorphicQuerySet

mkurek commented:
:ok:

ar4s commented:
:ok:
